src/widgets/base.vue

```diff
@@ -82,7 +82,7 @@ export default {
 			return typeof this.uiTab === 'number' ? this.uiTab : this.tab
 		},
 		uiTitle: function() {
-			return this.ui['title'] || this.schema['title'] || this.property
+			return this.ui['title'] ? (typeof this.ui['title'] == 'boolean' ? '' : this.ui['title']) : (this.schema['title'] || this.property)
```

Comment:
This is quite hard to read

Reply:
indeed.. must have been the evil-mini-me-of-maintenance. :(

Reply:
reverted